<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VariablesWidget</class>
 <widget class="QWidget" name="VariablesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableWidget" name="Variables">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="toolsLayout">
     <item>
      <widget class="QLineEdit" name="searchEdit">
       <property name="placeholderText">
        <string>Search</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="rightSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="removeButton">
       <property name="icon">
        <iconset theme="list-remove">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="addButton">
       <property name="icon">
        <iconset theme="list-add">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>VariablesWidget</receiver>
   <slot>AddButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>469</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeButton</sender>
   <signal>clicked()</signal>
   <receiver>VariablesWidget</receiver>
   <slot>RemoveButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Variables</sender>
   <signal>itemChanged(QTableWidgetItem*)</signal>
   <receiver>VariablesWidget</receiver>
   <slot>ListItemChanged(QTableWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>813</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>VariablesWidget</receiver>
   <slot>SearchEditChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>417</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>AddButtonClicked()</slot>
  <slot>RemoveButtonClicked()</slot>
  <slot>ListItemChanged(QTableWidgetItem*)</slot>
  <slot>SearchEditChanged(QString)</slot>
 </slots>
</ui>
